File: chatapp/app/src/main/java/com/example/chatapp/MainActivity.java
package com.example.chatapp;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {
    Button login,Register;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        login=findViewById(R.id.login);
        Register=findViewById(R.id.Register);

        login.setOnClickListener(new view.onClickLIstener()){
            public void onClick(View view){
                startActivity(new Intent(MainActivity.this,loginActivity.class));

            }


        }
        Register.setOnClickListener(new view.onClickLIstener(){
            public void onClick(View view){
                startActivity(new Intent(MainActivity.this,RegisterActivity.class));
            }
        });

    }
}

File: Myfirstapp/app/src/main/java/com/example/myfirstapp/MainActivity.kt
package com.example.myfirstapp

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        setSupportActionBar(toolbar)
        BigButton.setOnClickListener{a view!: d(tag: "daniel", msg "button was pressed")
        }
    }
}
